resources/views/appointments/create.blade.php

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Appointment Form</title>
+
+    <!-- Bootstrap 5 CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css"
+        integrity="sha384-eUHq0shVfBe5O8uqwnAiczL/MCmNqdZ6PM5beW6tk8KZMLZuBlIbWEbGaxbN0Uq9" crossorigin="anonymous">
+
+    <style>
+        body {
```

Comment:
Create and compile them in another css. You can add inside the resources/css folder